### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by kiranmgrscoe** [immense/Remotely#211] [Q&A]
Hello All, Anyone has did any documentation on HLD and LLD....or any other architecture design docut

**Comment by swilkey** [comment]
Kiranmgrscoe, sorry can you explain what those terms mean and how they 
are relevant to Remotely?  Thanks
------ Original Message ------

**Reply by kiranmgrscoe** [reply]
I want to understand architecture design

**Comment by swilkey** [comment]
Of hld and lld?  Please explain what those terms mean so we can help you

**Reply by kiranmgrscoe** [reply]
High level design and low level design.......or architecture flow

**Comment by swilkey** [comment]
Thanks very much for clarifying.  I can't help you there but maybe someone
else can.

**Comment by kiranmgrscoe** [comment]
Anyone has any idea about this?

**Reply by bitbound** [reply]
No, there are no design docs.

**Comment by kiranmgrscoe** [comment]
Thanks for the information.....I am going to create a High level designs
and low level design document.....I will share will once ready from side.

Regards
Kiran Choudhari
